_book/05-toolkit-reference/04-toolkit-options.md

```diff
@@ -15,10 +15,21 @@ see-also:
       - "/toolkit-reference/output-formats.html"
     transpose:
       - "/advanced-topics/transposition.html"
+    pageHeight:
+      - "/advanced-topics/controlling-the-svg-output.html"
+    pageWidth:
+      - "/advanced-topics/controlling-the-svg-output.html"
     resourcePath:
       - "/toolkit-reference/toolkit-methods.html#setresourcepath"
       - "/installing-or-building-from-sources/python.html#building-the-toolkit"
-    
+    scale:
+      - "/advanced-topics/controlling-the-svg-output.html#scaling"
+    scaleToPageSize:
+      - "/advanced-topics/controlling-the-svg-output.html#scaling"
+    smuflTextFont:
+      - "/advanced-topics/smufl.html#music-symbols-in-text"
+    unit:
+      - "/advanced-topics/controlling-the-svg-output.html"   
```

Comment:
should this link to the MEI unit section?

Reply:
I could not find one...

Reply:
it would probably be something like `/advanced-topics/controlling-the-svg-output.html#mei-unit`?

Reply:
Yes, I tried that but the the sub-sections are not listed in the table of content, so it does not work. I would need to change that. I think we can leave it for now. Just a bit more reading for the users, which is probably not bad ;-)

Reply:
ok